integration-tests/tests/fixtures/package_instance_fixtures.py

```diff
@@ -0,0 +1,56 @@
+"""Fixtures that start and stop CLP package instances for integration tests."""
+
+import logging
+import subprocess
+from collections.abc import Iterator
+
+import pytest
+
+from tests.utils.clp_mode_utils import CLP_MODE_CONFIGS
+from tests.utils.config import (
+    PackageConfig,
+    PackageInstance,
+)
+from tests.utils.package_utils import (
+    start_clp_package,
+    stop_clp_package,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def clp_package(
+    clp_config: PackageConfig,
+) -> Iterator[PackageInstance]:
```

Comment:
```suggestion
def clp_package_instance(
    clp_package_config: ClpPackageConfig,
) -> Iterator[ClpPackageInstance]:
```
ditto about fixture naming